Source/UI/IdentityUI.swift

```diff
@@ -462,6 +468,21 @@ extension IdentityUI {
         case let .byRoute(route, vc):
             handleRouteForUnpresentedUI(route: route, byPresentingIn: vc, client: client, completion: completion)
         case let .byLoginMethod(loginMethod, vc, localizedTeaserText, scopes):
+            if #available(iOS 13.0, *), case .sharedWebCredentials = loginMethod {
+                let sharedWebCredentialsController = SharedWebCredentialsController()
+                sharedWebCredentialsController.didCompleteWithError = { [weak self] _ in
+                    self?.show(input: .byLoginMethod(.password, presentingViewController: vc, localizedTeaserText: localizedTeaserText, scopes: scopes), client: client, completion: completion)
+                }
+                sharedWebCredentialsController.didCompleteWithAuthorization = { [weak self] user, password in
+                    guard let email = EmailAddress(user) else { return }
+                    self?._identityManager.login(username: .email(email), password: password, scopes: scopes, persistUser: true, useSharedWebCredentials: false, completion: { _ in
+                        completion(.skip)
+                    })
+                }
+                self.sharedWebCredentialsController = sharedWebCredentialsController
```

Comment:
Note: `authorizationController` has to be reset to avoid the retain cycle.

Reply:
What does "reset" mean in this context? You mean we have to explicitly set it to nil?